### PR TITLE
Xot 1185 remove market name prefix

### PR DESCRIPTION
**Pull request by jeanettemariemurphy**
…th 'the' alphabetically by country, without changing the display name

To do (delete all that do not apply):

 - [x] Change has a jira ticket that has the correct status.
 - [x] Changelog entry added.

### CODE REVIEW

File: export_readiness/serializers.py [hunk 1]
```diff
@@ -504,6 +504,10 @@ class ChildCountryGuidePageSerializer(BasePageSerializer):
     hero_image_thumbnail = wagtail_fields.ImageRenditionField('fill-640x360', source='hero_image')
     heading = serializers.CharField()
     sub_heading = serializers.CharField()
+    sorted_title = serializers.SerializerMethodField()
+
+    def get_sorted_title(self, parent):
+        return parent.title.lower().replace('the ', '')
```
Comment (richtier) [inline export_readiness/serializers.py]: similar to https://en.wikipedia.org/wiki/Scunthorpe_problem

this will have unwanted side effects for words ending in `the` for example:

`The British Pathe videos => british pa videos`

This would affect the following words: https://www.wordfind.com/ends-with/the/

Also `lower()` would have unwanted effects for acronyms:

`The CIA worldbook => cia worldbook`

regex can solve both :)

```
return re.sub(r'^the', '', parent.title, flags=re.IGNORECASE)
```

Comment (jeanettemariemurphy) [reply export_readiness/serializers.py]: @richtier this change does not affect the display name, as this is still using the 'title' field.  'sorted_title' is just for sorting on country names, so The Netherlands will still display on the UI as The Netherlands, but is alphabetically sitting under N.
The lower case was just to prevent a sort deprioritorising if the cms returned The netherlands, for example
Using a regex would still be a good idea, but i wasn;t sure if it was necessary in this case?

Comment (eadpearce) [reply export_readiness/serializers.py]: off the top of my head I don't know if any country name ends in 'the' BUT if there are some like this maybe it'd be better to only look for it at the start of the string

Comment (jeanettemariemurphy) [reply export_readiness/serializers.py]: happy to change to the regex, but the point being that even if a country did end in 'the', it wouldn't affect the logic of what is trying to be achieved, ie the last 3 letters would be removed from the word , which would still place them correctly alphabetical order. And the UI display would not be affect at all.

Comment (eadpearce) [reply export_readiness/serializers.py]: yeah actually it's probably too much of an edge case to be an issue

File: tests/export_readiness/test_serializers.py [hunk 2]
```diff
@@ -94,6 +94,29 @@ def test_country_guide_page_serializer(root_page, rf):
...
+        title='The Netherlands',
+        slug='country',
+        parent=markets_page)
+
+    serializer = ChildCountryGuidePageSerializer(
+        instance=country_guide,
+        context={'request': rf.get('/')}
+    )
+
+    sorted = serializer.data['sorted_title']
```
Comment (eadpearce) [inline tests/export_readiness/test_serializers.py]: `sorted` is a builtin python function. can you rename to something like `sorted_title`? https://docs.python.org/3.5/library/functions.html#sorted
